src/app/(protected)/analises/pessoas/consultar/process.tsx

```diff
@@ -83,13 +83,20 @@ const MemoizedCard = memo(Card);
 export function ProcessFinished({ analysis_info, document }: Props) {
   const modal = useModal();
 
+  const [haveProcess, setHaveProcess] = useState(false)
```

Comment:
Ai não precisaria desse estado
```suggestion
  const [haveProcess, setHaveProcess] = useState(false)
```